core/client/hdfs/src/main/java/alluxio/hadoop/HadoopUtils.java

```diff
@@ -32,6 +34,18 @@
 @ThreadSafe
 public final class HadoopUtils {
   private static final Logger LOG = LoggerFactory.getLogger(HadoopUtils.class);
+  public static final String ZOOKEEPER_IDENTIFIER = "zk@";
+
+  /**
+   * Identifies whether a {@link URI} uri is an Alluxio on Zookeeper URI.
+   *
+   * @param uri the input uri
+   * @return whether the uri is Alluxio on Zookeeper URI
+   */
+  public static boolean isZookeeperUri(URI uri) {
```

Comment:
nothing in here is hadoop-specific - I don't think this belongs in `HadoopUtils`

Reply:
Removed to the AbstractFileSystem.class and add a TODO comment. Thanks

```
    // TODO(lu) make Alluxio URI capable of processing Alluxio on Zookeeper URI
    // and expose methods to get URI type and Zookeeper addresses.
```